README.md

```diff
@@ -1,5 +1,7 @@
 ## Muse
 
+[![Build Status](https://travis-ci.org/kachayev/muse.svg?branch=master)](https://travis-ci.org/kachayev/muse)
```

Comment:
:memo: This build status image will not display correctly until this project is set up in Travis CI.